rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import typescript from '@rollup/plugin-typescript';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import cleanup from 'rollup-plugin-cleanup';
+import styles from 'rollup-plugin-styles';
+import {string} from 'rollup-plugin-string';
+import alias from '@rollup/plugin-alias';
+import json from '@rollup/plugin-json';
+
+import readmePlugin from './build/readme.plugin.js';
+
+const rollup = {
+	input: {
+		options: './source/options.tsx',
+		background: './source/background.ts',
+		'refined-github': './source/refined-github.ts',
+		'resolve-conflicts': './source/resolve-conflicts.ts',
+	},
+	output: {
+		dir: 'distribution/assets',
+		// AssetFileNames: "[name]-[hash][extname]", // for CSS
+	},
+	plugins: [
+		json(), // TODO: Drop after https://github.com/refined-github/shorten-repo-url/issues/47
+		styles(),
+		string({
+			include: '**/*.gql',
+		}),
```

Comment:
This could use a GQL-specific plugin in the future, but for now it's 1:1 for easier upgrade.

- https://github.com/rollup/plugins/tree/master/packages/graphql
- #6686